Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -138,12 +138,40 @@ extension Option where Value: ExpressibleByArgument {
   ///   - name: A specification for what names are allowed for this flag.
   ///   - parsingStrategy: The behavior to use when looking for this option's value.
   ///   - help: Information about how to use this option.
+  @available(*, deprecated, message: "Use init(wrappedValue:name:parsing:help:completion:) instead.")
   public init(
     wrappedValue: Value,
     name: NameSpecification = .long,
     parsing parsingStrategy: SingleValueParsingStrategy = .next,
-    completion: CompletionKind? = nil,
-    help: ArgumentHelp? = nil
+    completion: CompletionKind,
+    help: ArgumentHelp
+  ) {
+    self.init(
+      name: name,
+      initial: wrappedValue,
+      parsingStrategy: parsingStrategy,
+      help: help,
+      completion: completion)
+  }
+
+  /// Creates a property with a default value provided by standard Swift default value syntax.
+  ///
+  /// This method is called to initialize an `Option` with a default value such as:
+  /// ```swift
+  /// @Option var foo: String = "bar"
+  /// ```
+  ///
+  /// - Parameters:
+  ///   - wrappedValue: A default value to use for this property, provided implicitly by the compiler during property wrapper initialization.
+  ///   - name: A specification for what names are allowed for this flag.
+  ///   - parsingStrategy: The behavior to use when looking for this option's value.
+  ///   - help: Information about how to use this option.
```

Comment:
Could you add a description of the `completion` parameter?

Reply:
Sure!

Reply:
I noticed that a lot of `completionKind` parameters from other methods are not documented. What should we do about those ?

Reply:
Probably worth doing in a separate pull request (if you're up to it)